### PR TITLE
Feature/set up

### PR DESCRIPTION
**Pull request by lRebornsl**
# 🎏   In this *Milestone*  🙋‍♂️⁉:
✅ Create a new Rails app called 'hello-rails-react'.
✅ Use the command rails new -j webpack my_new_rails_app.
✅ Install needed node dependencies.
✅ Create a babel.config.js in the root folder of your project.
✅ Update webpack.config.js to include the babel loader.
✅ Make sure that your project has a Postgres database set up.
✅ Create a table for storing your messages and create 5 different greetings.
✅ Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and Rails route).
✅ Create a static view that will be the root of your app.
✅ Create the App component with react-router and render it in your static view.
✅ Create the Greeting component that will display a greeting. Set it up as a route in your App component.
✅ Create a store, an action, and a reducer that will connect to your API endpoint to get the random greeting.
✅ Display the random greeting in your Greeting component.
✅ No linters error.
✅ Follow the best GitHub flow.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "react": "^18.1.0",
+    "react-dom": "^18.1.0",
+    "react-redux": "^8.1.3",
+    "react-router-dom": "^6.18.0"
+  },
+  "scripts": {
+    "build": "esbuild app/javascript/*.* --bundle --sourcemap --outdir=app/assets/builds --loader:.js=jsx"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.23.2",
```
Comment (mukhammadrizooff) [inline package.json:16]: - [ ] [REQUIRED] Good job so far! Please required config files should be set up as showed in [the totorial](https://betterprogramming.pub/react-with-rails-2022-bd28e1fcd355#ad0a): and this file created at your working place 🙏🏼 

`babel.config.js`

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @lRebornsl, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (lRebornsl) [reply package.json:16]: Thanks for your feedback 👍🏼 

Solved 😄

**Review by IngDavidTM — APPROVED** [review]
Hi @lRebornsl 🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitFlow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

- [OPTIONAL] Every microverse project must include a well-crafted README file, so please make sure to include one.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**